### PR TITLE
Remove LooReeeE

### PR DESCRIPTION
**Pull request by ghambhackmud**
literally anything

### CODE REVIEW

File: wiki/lore.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ tags: [lore]
 
 ---
 
-LooooRrreeEE
\ No newline at end of file
+This is lore
```
Comment (danswann) [inline wiki/lore.md:11]: Great idea!

**Review by danswann — COMMENTED** [review]
Very good change.

**Comment by 20k** [comment]
hello i am a comment

File: wiki/lore.md (hunk 1 above)
Comment (20k) [reply wiki/lore.md:11]: perfect

**Comment by ast-ral** [comment]
(for future reference, this is just a testing PR)

**Review by 20k — COMMENTED** [review]
i am a fish

File: wiki/lore.md [hunk 2]
```diff
@@ -8,4 +8,4 @@ tags: [lore]
 
 ---
 
-LooooRrreeEE
\ No newline at end of file
+This is lore.
```
Comment (danswann) [inline wiki/lore.md:11]: I suggest an ! instead.

**Review by danswann — CHANGES_REQUESTED** [review]

**Pull request merged**